indico/MaKaC/webinterface/pages/conferences.py

```diff
@@ -5852,8 +5851,9 @@ def _getHeadContent( self ):
 
         if self._selectedCSS:
             printCSS = printCSS + """<link rel="stylesheet" type="text/css" href="%s" >"""%self._selectedCSS.getURL()
-        elif self._styleMgr.getCSS():
-            printCSS = printCSS + """<link rel="stylesheet" type="text/css" href="%s" >"""%self._styleMgr.getCSS().getURL()
+        if self.css_url:
+            link = '<link rel="stylesheet" type="text/css" href="{url}">'.format(url=self.css_url)
```

Comment:
because I couldn't manage to put everything in one line otherwise! But after we worked on it, now it fits ;)